perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -83,6 +84,20 @@
                      'Allow swiftclient to access Swift service without \n'
                      'having to verify the SSL certificate')
 
+flags.DEFINE_integer('objects_per_stream', 1000,
+                     'Number of objects to send and/or receive per stream. '
+                     'Only applies to the api_multistream scenario.',
+                     lower_bound=1)
+flags.DEFINE_integer('object_size_kb', 1,
```

Comment:
Currently I see only the SingleStream test that will likely use this object_size flag. Other test scenarios (namespace, one byte RW, or cli) all have their own object sizes. If we call it object_storage_object_size, people would think we'd use this object size for all scenarios, which is not true. Consider a different name...

Alternatively, since you are going to make a second PR to support object size distribution very soon, I am leaning towards not introduce this new flag that specifies a single value object size. Once you add support for object size distribution, then it's very easy to specify a single size via the distribution flag, right? (something like "200MiB, 100%")